modules/plot/test/spec.coffee

```diff
@@ -243,15 +254,24 @@ describe "plot", ->
       checkOptionAndSetterGetter(hx.BarSeries, 'group', ['group1', 'group2'])
 
     describe "render", ->
-      it 'should display when the height is positive', ->
-        selection = hx.detached('div').style('height', '400px')
-        hx.select('body').add(selection)
-        graph = new hx.Graph(selection.node())
-        axis = graph.addAxis('linear', 'linear')
-        a1 = axis.addSeries()
-        graph.render()
-        selection.selectAll('.hx-series').size().should.equal(2)
-        selection.remove()
+      it 'should display when the height is positive', (done) ->
+        graphOpts =
+          axes: [
+            series: [['line']]
```

Comment:
The series option for an axis and axes option for a graph also exist for graphs constructed with a constructor.